readme.md

```diff
@@ -1,19 +1,29 @@
 # GitHub Actions Git
 
-A collection of GitHub actions for Git projects.
+A collection of GitHub actions.
 
-## Example Playbook
+## Create or Update Git Tag
+This action will crate a new tag. If the tag already exists it will be overwritten (requires force-push).  
```

Comment:
```suggestion
## Create or Update Git Tag

This action will crate a new tag. If the tag already exists it will be overwritten (requires force-push).  
```